test/hotspot/jtreg/compiler/loopopts/superword/TestCompatibleUseDefTypeSize.java

```diff
@@ -473,4 +505,201 @@ static Object[] test11(char[] a) {
         }
         return new Object[]{ a, new char[] { m } };
     }
+
+    // Narrowing
+
+    @Test
+    @IR(applyIfCPUFeature = { "avx", "true" },
+        applyIfOr = {"AlignVector", "false", "UseCompactObjectHeaders", "false"},
```

Comment:
Do you think these would be supported with `asimd` as well?
If you just cannot test with it feel free to file an RFE and then I can find someone to take care of it (e.g. as a starter bug).